config/prow/config.yaml

```diff
@@ -226,3 +226,49 @@ presets:
   env:
     - name: GOPPC64
       value: power9
+# knative powervs preset
+- labels:
+    preset-knative-powervs: "true"
+  env:
+    - name: IBMCLOUD_API_KEY
+      valueFrom:
+        secretKeyRef:
+          name: knative-apikey
+          key: apikey
+    - name: GITHUB_TOKEN
+      valueFrom:
+        secretKeyRef:
+          name: github-token
+          key: token
+    - name: KO_FLAGS
+      value: --platform=linux/ppc64le
+    - name: PLATFORM
+      value: linux/ppc64le
+    - name: KO_DOCKER_REPO
+      value: na.artifactory.swg-devops.com/sys-linux-power-team-ftp3distro-docker-images-docker-local/knative
+    - name: DOCKER_CONFIG
+      value: /root/.docker
+
+  volumeMounts:
+    - name: git-volume
+      mountPath: /opt/cluster/
+    - name: ssh-key-volume
+      mountPath: /root/.ssh/
+      readOnly: true
+    - name: config-json-volume
+      mountPath: /root/.docker/
+      readOnly: true
+
+  volumes:
+    - name: git-volume
+      secret:
+        defaultMode: 256 
+        secretName: github-token
+    - name: ssh-key-volume
+      secret:
+        secretName: ssh-key-secret
+        defaultMode: 384 
+    - name: config-json-volume
+      secret:
+        secretName: config-json-secret
+        defaultMode: 420 
```

Comment:
Get rid of this ` no newline at end of file`

Reply:
done